keen/explorer
# Fixes #139: "Exists" filter breaks when changing the property name

## What does this PR do?
It fixes #139. It ensures that if you change the property name on a filter with an `exists` oprerator, nothing else changes and you can still run the filter.

## How can I test this?
1) Create a query with a filter that has any property name, an exists operator, and either true or false for the value.
2) Run the query.
3) Change that filter's property name to something else
4) Run the query again.
5) Confirm it ran successfully.
